tests/common.go

```diff
@@ -4950,6 +4950,34 @@ func DumpKubeconfigs(kubeconfigList []string) {
 	dash.VerifyFatal(err, nil, fmt.Sprintf("verfiy getting kubeconfigs [%v] from configmap [%s]", kubeconfigList, configMapName))
 }
 
+// DumpCloudconfigs gets kubeconfigs from configmap
+func DumpCloudconfig(cloudConfigName string) error {
+	err := dumpCloudConfigs(cloudConfigName)
+	if err != nil {
+		return fmt.Errorf("Failed to get cloudconfig")
+	}
+	return nil
+}
```

Comment:
Why do we need this function? Please optimise the code.

Reply:
done